packages/react-day-picker/src/contexts/Focus/FocusContext.tsx

```diff
@@ -59,35 +96,148 @@ export function FocusProvider({
   const focusDayBefore = () => {
     if (!focusedDay) return;
     const before = addDays(focusedDay, -1);
-    setDay(before);
+    changeInitialFocus(before);
```

Comment:
Please remove these here and below:

```suggestion
```

Reply:
These are functions that implement the reset of the initial focus and their deletion will result in broken behavior.